domains/certificates/Shared/CertificateEvents/ProductionCertificateTransferred.cs

```diff
@@ -6,12 +6,5 @@ namespace CertificateEvents;
 public record ProductionCertificateTransferred(
     Guid CertificateId,
     string Source,
-    string Target,
-
-    // Duplicated values from below
-    string GridArea,
-    Period Period,
-    Technology Technology,
-    ShieldedValue<string> ShieldedGSRN,
-    ShieldedValue<long> ShieldedQuantity
+    string Target
```

Comment:
As a consequence of deleteing the duplicate values here, we should also remove the private fields in domains/certificates/Shared/CertificateEvents/Aggregates/ProductionCertificate.cs (which is also causing codesmells in Sonarcloud)